bin/ansible2gdeploy.py

```diff
@@ -104,15 +160,25 @@ def main():
     print("clients: " + ", ".join(clients), file=sys.stderr)
     print("devices: " + ", ".join(storage_devices), file=sys.stderr)
 
+    host_transformation = {
+        'fqdn': host_fqdn,
+        'short': host_short,
+        'ip': host_ip,
+        'mixed': host_mixed,
+        'random': host_random,
+        }
+
     # update gdeploy config files
     for gdeploy_conf_file, gdeploy_conf in gdeploy_confs.items():
         # ignore hosts section if present
         if gdeploy_conf.has_section("hosts"):
             gdeploy_conf.remove_section("hosts")
         # add servers into hosts sections
         gdeploy_conf.add_section("hosts")
+        print("hosts_definition: %s" % args.hosts_definition, file=sys.stderr)
         for server in servers:
-            gdeploy_conf.set("hosts", server, None)
+            gdeploy_conf.set("hosts", \
```

Comment:
Nitpicking: why is the `\` at the end of the line?

Reply:
@mbukatov because of pylint complains:
```
C0330: l181: [(bad-continuation), ] Wrong continued indentation (add 9 spaces).
                    host_transformation[args.hosts_definition](server), None)
                    ^        |
```
Adding additional 9 spaces will be definitely more ugly, because the first argument for the method (`"hosts"`) is short, but the second one is much longer.

Reply:
ok, you are right...